### PR TITLE
Code complete

### PR DESCRIPTION
**Pull request by Justin-Locke**

### CODE REVIEW

File: fishingBoat.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { fishCatch } = require("./fishCatch.js")
```
Comment (gcziprusz) [inline fishingBoat.js:1]: Well done separating the data from the boat logic!

**Comment by gcziprusz** [comment]
Please make sure that `npm run test` passes, it likely only needs some updates to your outputs html formatting 
```
 PASS  test/boat.test.js
  Fishing boat
    ✓ Can view all inventory (4ms)

 PASS  test/monger.test.js
  Fishmonger
    ✓ Can view all inventory (3ms)
    ✓ Can view inventory that cost less than $6 per fish (1ms)
    ✓ Can view inventory that cost less than $4 per fish

 FAIL  test/restaurant.test.js
  Restaurant
    ✕ Producing the menu (5ms)

  ● Restaurant › Producing the menu

    expect(received).toBe(expected) // Object.is equality

    Expected: 232
    Received: 286

       5 |     test("Producing the menu", () => {
       6 |         expect(fishMenu(3.49).length)
    >  7 |             .toBe(`<h1>Menu</h1>
         |              ^
       8 | <article class="menu">
       9 | <h2>Salmon</h2>
      10 |     <section class="menu__item">Salmon Soup</section>

      at Object.<anonymous> (test/restaurant.test.js:7:14)

Test Suites: 1 failed, 2 passed, 3 total
Tests:       1 failed, 4 passed, 5 total
Snapshots:   0 total
Time:        1.057s
Ran all test suites.

Watch Usage
 › Press f to run only failed tests.
 › Press o to only run tests related to changed files.
 › Press p to filter by a filename regex pattern.
 › Press t to filter by a test name regex pattern.
 › Press q to quit watch mode.
 › Press Enter to trigger a test run.
```

**Comment by Justin-Locke** [comment]
Just removed the added spaces from my HTML formatting that was causing the last test to fail. Thank you!

**Review by gcziprusz — APPROVED** [review]
Well done!

**Pull request merged**